### DISCUSSION TITLE
Welcome to Textual Discussions!

### DISCUSSION THREAD

**Discussion by willmcgugan** [Textualize/textual#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dissolved** [comment]
Is this the primary place for Textual community? Or is there a Slack space or other more interactive community. I'm excited about Textual, and if I can find the time would love to pitch in to help the effort. I'm still getting my bearings. Thanks!

Oh, and for introductions. I'm Ryan Sandridge. I've been writing code for 30+ years, but only been doing Python the past 5ish. I am tired of web apps for everything, and have a particular interest in developer tooling, and Textual seems to be off to a great start. I work currently for Arcadia and we have internal needs for better developer tooling, and I'm very interested in using Textual for that.
